docs/fields.md

```diff
@@ -355,6 +355,20 @@ DateTimeField supports int, float, string (isoformat), date object and of course
 !!! Note
     `auto_now` and `auto_now_add` set the `read_only` flag by default. You can explicitly set `read_only` to `False` to be still able to update the field manually.
 
+#### DurationField
+
+```python
+import datetime
+import edgy
+
+
+class MyModel(edgy.Model):
+    worked: datetime.timedelta = edgy.DurationField()
+    ...
+
+```
+
```

Comment:
extra space not needed.

Can we add an explanation of the reason for the duration field and cases where it can be used?